src/libpriv/rpmostree-origin.cxx

```diff
@@ -116,22 +116,33 @@ rpmostree_origin_parse_keyfile (GKeyFile         *origin,
   ret->cached_unconfigured_state = g_key_file_get_string (ret->kf, "origin", "unconfigured-state", NULL);
 
   g_autofree char *refspec = g_key_file_get_string (ret->kf, "origin", "refspec", NULL);
+  g_autofree char *imgref = g_key_file_get_string (ret->kf, "origin", "imgref", NULL);
```

Comment:
This is total bikeshed on naming but...perhaps `from` in homage to `Dockerfile`'s [FROM](https://docs.docker.com/engine/reference/builder/#from)?

Reply:
At first glance I thought `from` would be quite unintuitive as someone seeing it would have a hard time guessing what it meant.
Thinking about it more though, since it is always followed by something that's obviously an "image reference", maybe it's not so bad and `from` will help with adding the derivation story later on :)